[2023-08-31T03:55:27.096+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: process_sale_data.validate_result manual__2023-08-31T03:55:20.596452+00:00 [queued]>
[2023-08-31T03:55:27.106+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: process_sale_data.validate_result manual__2023-08-31T03:55:20.596452+00:00 [queued]>
[2023-08-31T03:55:27.107+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 1
[2023-08-31T03:55:27.121+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): validate_result> on 2023-08-31 03:55:20.596452+00:00
[2023-08-31T03:55:27.125+0000] {standard_task_runner.py:57} INFO - Started process 5831 to run task
[2023-08-31T03:55:27.128+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'process_sale_data', 'validate_result', 'manual__2023-08-31T03:55:20.596452+00:00', '--job-id', '178', '--raw', '--subdir', 'DAGS_FOLDER/dags.py', '--cfg-path', '/tmp/tmpzgi43959']
[2023-08-31T03:55:27.130+0000] {standard_task_runner.py:85} INFO - Job 178: Subtask validate_result
[2023-08-31T03:55:27.176+0000] {task_command.py:410} INFO - Running <TaskInstance: process_sale_data.validate_result manual__2023-08-31T03:55:20.596452+00:00 [running]> on host 888526fa4a36
[2023-08-31T03:55:27.249+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='process_sale_data' AIRFLOW_CTX_TASK_ID='validate_result' AIRFLOW_CTX_EXECUTION_DATE='2023-08-31T03:55:20.596452+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-08-31T03:55:20.596452+00:00'
[2023-08-31T03:55:27.379+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-31T03:55:27.385+0000] {sql.py:375} INFO - Running statement: SELECT COUNT(*) FROM consumption_alcoholic_20230831, parameters: None
[2023-08-31T03:55:27.386+0000] {sql.py:384} INFO - Rows affected: 1
[2023-08-31T03:55:27.391+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-31T03:55:27.394+0000] {sql.py:375} INFO - Running statement: SELECT COUNT(*) FROM consumption_cereals_bakery_20230831, parameters: None
[2023-08-31T03:55:27.395+0000] {sql.py:384} INFO - Rows affected: 1
[2023-08-31T03:55:27.401+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2023-08-31T03:55:27.404+0000] {sql.py:375} INFO - Running statement: SELECT COUNT(*) FROM consumption_meats_poultry_20230831, parameters: None
[2023-08-31T03:55:27.405+0000] {sql.py:384} INFO - Rows affected: 1
[2023-08-31T03:55:27.430+0000] {logging_mixin.py:149} INFO - Import data successful
[2023-08-31T03:55:27.430+0000] {python.py:183} INFO - Done. Returned value was: None
[2023-08-31T03:55:27.440+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=process_sale_data, task_id=validate_result, execution_date=20230831T035520, start_date=20230831T035527, end_date=20230831T035527
[2023-08-31T03:55:27.462+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2023-08-31T03:55:27.480+0000] {taskinstance.py:2653} INFO - 1 downstream tasks scheduled from follow-on schedule check
